src/components/Documentation/Layout/JsonSchemas/JsonSchemasComponent.module.scss

```diff
@@ -0,0 +1,17 @@
+.doc-main-title {
+        margin-bottom: 40px;
+        font-size: 48px;
+        text-align: center;
+        color: #333;
+};
```

Comment:
why semicolon here?

Reply:
removed this, was repeated, you are right